pkgs/tools/security/sudolikeaboss/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fixDarwinDylibNames, darwin }:
+buildGoPackage rec {
+  name = "sudolikeaboss-${version}";
+  version = "0.2.1";
+
+  goPackagePath = "github.com/ravenac95/sudolikeaboss";
+  src = fetchFromGitHub {
+    owner = "ravenac95";
+    repo = "sudolikeaboss";
+    rev = "v${version}";
+    sha256 = "1zsmy67d334nax76sq0g2sczp4zi19d94d3xfwgadzk7sxvw1z0m";
+  };
+  goDeps = ./deps.nix;
+
+  propagatedBuildInputs = with darwin.apple_sdk.frameworks; [
+    Cocoa
+    fixDarwinDylibNames
+  ];
+
+  postInstall = ''
+    install_name_tool -delete_rpath $out/lib -add_rpath $bin $bin/bin/sudolikeaboss
+  '';
+
+  meta = with stdenv.lib; {
+    inherit version;
+    homepage = https://github.com/ravenac95/sudolikeaboss;
```

Comment:
You could use `inherit (src.meta) homepage;` since you're using `fetchFromGitHub` :wink: